src/dlgConnectionProfiles.cpp

```diff
@@ -2337,3 +2328,162 @@ QIcon dlgConnectionProfiles::customIcon(const QString& text) const
     }
     return QIcon(background);
 }
+
+void dlgConnectionProfiles::clearNotificationArea()
+{
+    notificationArea->hide();
+    notificationAreaIconLabelWarning->hide();
+    notificationAreaIconLabelError->hide();
+    notificationAreaIconLabelInformation->hide();
+    notificationAreaMessageBox->clear();
+}
+
+void dlgConnectionProfiles::slot_searchTimerTimeOut()
+{
+    reenableAllProfileItems();
+    if (notificationAreaMessageBox->text() == tr("As you type more letters in the name of a profile all the non-matching ones "
```

Comment:
It's intuitive enough to work with that it needs no explanation - and it's also impossible to read it fast enough :)

Reply:
Guess so - I must confess that is one incentive to drag the time interval out... :smiling_imp: ... but I'll leave the refactoring into `clearNotificationArea()` as that is still useful I think.

Reply:
Yes agree, `clearNotificationArea()` is good!

Reply:
clearNotificationArea ist super good - do more of those! 🌟